tracer/src/Datadog.Trace/AppSec/Waf/Waf.cs

```diff
@@ -91,6 +91,40 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+        public bool UpdateRules(IEnumerable<RuleData[]> res)
```

Comment:
This belongs to the blocking PR?

Reply:
I think it makes sense to have it here since it's a feature of the waf 1.5 to be able to give it rule datas, independently of blocking the request. It surely comes from a model of RCM but it doesnt have to be coupled to it

Reply:
Fair enough.